src/deps.txt

```diff
@@ -57,3 +57,6 @@ python3-flake8 python3-pytest python3-pytest-cov
 
 # Support for Koji uploads.
 krb5-libs krb5-workstation koji-utils python3-koji python3-koji-cli-plugins
+
+# For signing
```

Comment:
`For sending signing requests` maybe since we don't use fedora messaging for actual signing :)